src/main/java/com/google/cloud/genomics/utils/Paginator.java

```diff
@@ -94,9 +94,9 @@
  *String datasetId = ...;
  *Paginator.Readsets searchReadsets = Paginator.Readsets.create(stub);
  *for (Readset readset =
- *    searchReadsets.search(new SearchReadsetsRequest().setDatasetId(datasetId))) {
+ *    searchReadsets.search(new SearchReadsetsRequest().setDatasetId(datasetId))) &#123;
  *  // do something with readset
- *}
+ *&#125;
```

Comment:
I checked out [stackoverflow](http://stackoverflow.com/questions/647195/how-do-you-escape-curly-braces-in-javadoc-inline-tags-such-as-the-code-tag) for a bit of background information, and man is this messed up.
 * `<code>` requires you to escape angle brackets
 * `{@code ... }` requires you to escape braces